.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": ["@svitejs/changesets-changelog-github-compact", { "repo": "sveltejs/svelte" }],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "baseBranch": "master",
+  "updateInternalDependencies": "patch",
```

Comment:
while we don't have any right now, it would be good to align the config for dependencies with kit https://github.com/sveltejs/kit/blob/master/.changeset/config.json  (this reduces the amount of bumping with ^ workspace versions)